.github/workflows/create-release.yml

```diff
@@ -0,0 +1,73 @@
+name: Create Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      versionName:
```

Comment:
so this version might be used for both delphi_utils and the indicators? if I specify patch then indicators will be patched in any case and delphi_utils might

Reply:
That's correct. The indicators version helps us track when particular changes went into production, and that includes changes to utils.

Intended use is to deploy whenever a change either in utils or indicators needs to go out to prod. Initial version selection (patch/minor/etc) would be made based on that change. If we're trying to put out a patch change to indicators but it turns out there's a change to utils as well, the PR text alerts the change. If the utils change is also a patch, fine; if it merits a minor version bump, then the PR can be closed and the workflow re-run as minor.